two-ff-suggests/index.html

```diff
@@ -43,19 +39,21 @@
             sb1.onfocus = onfocusSuggest1;
             sb2.onfocus = onfocusSuggest2;
 
-            // remove the suggest from the data flow so it does not receive data anymore
+            // Remove the suggest from the data flow so it does not receive data anymore
             function unsubscribeSuggest(suggestElement) {
-                // check if element is already listening
+                // Check if element is already listening
+                // Be careful, "_suggestSubscriptionKey" is an internal property of the component and may be a subject of change in future releases
```

Comment:
Would phrase it a bit more dramatically to even more emphasize the dangers of using internal stuff. And to make it more of an eye-catcher.
```suggestion
                // CAUTION! "_suggestSubscriptionKey" is an internal property of the component and is subject to change without prior notice
```

Reply:
Changed - should be noticed now.